kubernetes/chart/zulip/Chart.yaml

```diff
@@ -15,26 +15,26 @@ version: 0.4.0
 appVersion: "6.1-0"
 dependencies:
   - name: memcached
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     tags:
       - memcached
-    version: 6.0.16
+    version: 6.3.5
   - name: rabbitmq
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     tags:
       - rabbitmq
-    version: 8.32.0
+    version: 11.9.1
```

Comment:
I guess since end users can choose whether or not to upgrade to our new charts, [upstream's notes about major changes in this upgrade path](https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/README.md#to-900) probably shouldn't block this? Noting that link anyway.